### ISSUE TITLE
Incoming webhooks not useful without notifying @all

### ISSUE THREAD

**Issue by antgel** [RocketChat/Rocket.Chat#12247]
I just set up my first Slack-compatible webhook, to integrate with New Relic. I have past experience of rolling my own webhooks, where it's easy to add @all to the message. But there's no way to do that with the `Script enabled: false` type of webhook. If I can't notify the entire channel when New Relic reports a problem in production it's kind of useless.

Therefore I suggest a toggle in the Incoming webhook admin, to notify all (or not).

**Comment by graywolf336** [comment]
Make sure the user it is posting as has permission to use `@all`.

**Comment by antgel** [comment]
@graywolf336 I tried that, by adding `Mention All` permission to the `bot` role. However, that doesn't make the incoming webhook notify `@all` i.e. the behaviour is the same as it was before.

**Comment by geekgonecrazy** [comment]
does the mention happen to be in an attachment instead of the text section?  I’ve used it several times in text section but in attachment it never triggers

**Comment by antgel** [comment]
Yes, I believe it is in the attachment section. The text section is less useful as it doesn't support rich text.

**Comment by antgel** [comment]
Hi RC team, this just bit us, does anyone have any interest in working on this? Webhooks that don't alert aren't that useful.

**Comment by antgel** [comment]
And again tonight. Anyone from the RC team interested in this? By the way @geekgonecrazy , looking at my original message, I think there are two different issues:

1. Mentions in HTML don't get parsed.
2. No way to tweak webhook to notify all in the case where we can't control the incoming text e.g.
![image](https://user-images.githubusercontent.com/295571/67528410-118c3380-f6c2-11e9-8c18-04dbe919a70c.png)

This is really about the second issue, because we rarely write custom webhook code these days. Fliipping title back. :)
